mu-plugins/central-hub/src/meta-data/meta-box.php

```diff
@@ -39,6 +40,8 @@ function register_meta_boxes() {
 			$config['priority'],
 			$config['callback_args']
 		);
+
+		return true;
```

Comment:
@rgadon107 Returning here means that it only processes 1 meta box. Why? The loop runs once and then returns immediately. This is not the behavior you want.

There's no need to return `true`. Why? Returning `true` implies that a meta box was properly registered with WordPress. But that does not correlate. Why? WordPress controls the registration itself. In your test, you use the global variable to validate that the expected box was actually registered. 

Therefore, `true` here does not tell you anything. It's unnecessary and misleading.

Reply:
@hellofromtonya I added the expression `return true` to `register_meta_box()` only because I needed some way to test an assertion that a valid meta box key was registered. But the you're right about the `return` control structure. Once that expression is reached, WP will bail out of the `foreach` loop, and no further processing would occur. 

Can I invoke the global variable `$wp_meta_box` in the unit test for `register_meta_boxes()? Would I do that with Brain Monkey, or by invoking the function `test_add_meta_box` from `/wordpress-develop/tests/phpunit/tests/admin/includesTemplate.php`?